Unix/samples/Providers/Indication/Lifecycle/XYZ_ComputerSystem.c

```diff
@@ -10,6 +10,7 @@
 /* @migen@ */
 #include <MI.h>
 #include <util.h>
+#include <pal/sem.h>
```

Comment:
The wrapper in pal/sem.h is very good, so we needn't change any method in code, one calling is here: https://github.com/Microsoft/omi/blob/master/Unix/samples/Providers/Indication/Lifecycle/XYZ_ComputerSystem.c#L226